sheet_test.go

```diff
@@ -344,3 +345,91 @@ func TestSetSheetName(t *testing.T) {
 	f.SetSheetName("Sheet1", "Sheet1")
 	assert.Equal(t, "Sheet1", f.GetSheetName(0))
 }
+
+func TestNewSheetWithRowNum(t *testing.T) {
+	f := NewFile()
+	f.NewSheetWithRowNum("sheet1", 100)
+	f.SetCellInt("sheet1", "A100", 100)
+	val, err := f.GetCellValue("sheet1", "A100")
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.Equal(t, "100", val)
+}
+
+func BenchmarkNewSheetWithRowNum(b *testing.B) {
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			newSheetWithRowNum()
+		}
+
+	})
+}
+func BenchmarkNewSheetWithOutRowNum(b *testing.B) {
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			newSheetWithOutRowNum()
+		}
+	})
+}
+func BenchmarkNewSheetWithStreamWriter(b *testing.B) {
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			newSheetWithStreamWriter()
+		}
+	})
+}
+func newSheetWithRowNum() {
+	file := NewFile()
+
+	file.NewSheetWithRowNum("sheet1", 1000)
+
+	for i := 0; i < 1000; i++ {
+		file.SetCellInt("sheet1", "A"+strconv.Itoa(i+1), i)
+	}
+	file = nil
```

Comment:
The sheet data doesn't save in `newSheetWithRowNum` and `newSheetWithOutRowNum`, but `newSheetWithStreamWriter`, so these profiling benchmarks are unfair comparison.

Reply:
sry.I didn't notice.